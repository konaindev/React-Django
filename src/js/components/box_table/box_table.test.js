import BoxTable from "./index";
import renderer from "react-test-renderer";

describe("BoxTable", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <BoxTable>
          <div>1</div>
          <div>2</div>
          <div>3</div>
        </BoxTable>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
