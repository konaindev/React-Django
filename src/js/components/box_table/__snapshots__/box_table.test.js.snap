// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoxTable renders correctly 1`] = `
Array [
  <div>
    1
  </div>,
  <div>
    2
  </div>,
  <div>
    3
  </div>,
]
`;
